integration-tests/scala/pom.xml

```diff
@@ -73,7 +73,6 @@
                 <configuration>
                     <systemPropertyVariables>
                         <!--suppress MavenModelInspection -->
-                        <maven.home>${maven.home}</maven.home>
                         <maven.repo>${settings.localRepository}</maven.repo>
```

Comment:
I wonder if we should move this one to the build-parent too. @aloubyansky WDYT?

Reply:
Shouldn't hurt!

/cc @aloubyansky

Reply:
Agreed.

Reply:
Done, also for `failsafe`.

I see no harm in adding this:
```xml
<project.version>${project.version}</project.version>
```
to build-parent (or IT parent) as well, but I did not touch it for now.